packages/internals/src/get-generators/getGenerators.ts

```diff
@@ -120,8 +119,6 @@ export async function getGenerators(options: GetGeneratorOptions): Promise<Gener
     throw new Error(missingDatasource)
   }
 
-  printConfigWarnings(config.warnings)
-
```

Comment:
Are the config warnings the same warnings as our lint warnings?
If so, we could check where it's used and remove it later